aix/generate_wazuh_packages.sh

```diff
@@ -32,11 +32,11 @@ show_help() {
   echo
   echo "Usage: $0 [OPTIONS]"
   echo
-  echo "    -e Install all the packages necessaries to build the RPM package"
-  echo "    -b <branch> Select Git branch. Example v3.5.0"
-  echo "    -s <rpm_directory> Directory to store the resulting RPM package. By default: /tmp/build"
-  echo "    -p <rpm_home> Installation path for the package. By default: /var"
-  echo "    -h Shows this help"
+  echo "    -b, --branch <branch>              Select Git branch or tag e.g. $BRANCH"
```

Comment:
Wrong indentation

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/219/commits/164434a1b72bd763d090f7356fec52718bf10d99